core/frontend/src/views/ExtensionManagerView.vue

```diff
@@ -192,15 +198,24 @@ export default Vue.extend({
         onDownloadProgress: (progressEvent) => {
           tracker.digestNewData(progressEvent)
           this.pull_output = tracker.pull_output
+          this.download_percentage = tracker.download_percentage
+          this.extraction_percentage = tracker.extraction_percentage
+          this.status_text = tracker.overall_status
         },
       })
         .then(() => {
-          this.show_dialog = false
           this.fetchInstalledExtensions()
         })
         .catch((error) => {
           notifier.pushBackError('EXTENSIONS_INSTALL_FAIL', error)
         })
+        .finally(() => {
+          this.show_dialog = false
+          this.pull_output = ''
+          this.download_percentage = 0
+          this.extraction_percentage = 0
+          this.status_text = ''
+        })
```

Comment:
note here that we are not going back to the initial state, as `download_total` and `extraction_total` are not being zeroed.

Reply:
these are not available in this context. plus we use a new instance of pulltracker for each pull, so things in there don't require re-zeroing